src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -803,8 +788,9 @@ public void verifyTablePattern(int tableNum, int column, String patternString) {
      * Verifies that the currently loaded page has the same HTML content as
      * the content given in the file at {@code filePath}. <br>
      * The HTML is checked for logical equivalence, not text equivalence.
-     * @param filePath If this starts with "/" (e.g., "/expected.html"), the
-     * folder is assumed to be {@link TestProperties.TEST_PAGES_FOLDER}.
+     * @param filePath
+     *         If this starts with "/" (e.g., "/expected.html"), the
```

Comment:
Re-quoting myself:
We have enough rules to make up a huge amount of technical debt. Let's not create another. This is not a big issue anyway.